webpack/plugins.js

```diff
@@ -1,12 +1,12 @@
-var config = require('config');
-var fs = require('fs');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
-var HtmlWebpackPlugin = require('html-webpack-plugin');
-var webpack = require('webpack');
+var config = require('config'); // eslint-disable-line no-var
```

Comment:
I tried once to use es6 in webpack config but it didn't work well at the time. If you know how to make it work, please do so